### DISCUSSION TITLE
A standalone Python extension "memimport"

### DISCUSSION THREAD

**Discussion by SeaHOH** [py2exe/py2exe#172] [Show and tell]
Happy New Year! Everybody!

I have created a repo [memimport](https://github.com/SeaHOH/memimport) just now, almost the codes are comes from here.
I just improved the `zipextimporter` module, which I want use it in the REPL.

But, I have a question: is there any problem if I release it to PyPI?

---

Edit: the name has been changed from `memimporter` to `memimport`.

**Comment by albertosottile** [comment]
Hi, thank you for your contribution. There is no problem at all, as long as you fulfill the terms of the two licenses of `py2exe`. I would encourage you to open PRs in the `py2exe` repo if you develop useful modifications on your side.

**Reply by SeaHOH** [reply]
Thank you for response!

I bound two licenses into `LICENSE.txt` ( also keep the origin files ), because I did not make a `memimport` or any other package, just two modules, and wheel will not include multiple license files in `dist-info` directory.

Yes, I will open PRs ( in a way suitable for `py2exe` ), after tests and debugged the errors. I am not familiar C and Win32 API, possibility never modify C sources, so I plan the version numbers to follow `py2exe`'s.
